cloudbuild.yaml

```diff
@@ -25,81 +28,299 @@ steps:
   - container
   - clusters
   - get-credentials
-  - "limani-integ"
+  - cluster-1
   - --region
-  - "us-central1"
+  - us-west1-a
   - --project
-  - "$PROJECT_ID"
+  - $PROJECT_ID
+  waitFor:
+  - '-'
 
-- id: Initialized
-  name: bash
+- id: Pull cloud-marketplace-tools deployer_envsubst Image
+  name: gcr.io/cloud-builders/docker
+  entrypoint: /bin/bash
+  args:
+  - -exc
+  - |
+    MARKETPLACE_TOOLS_TAG=sha_7c70bcd83a8c490a11f93abfc4ec7a5f39c61ab6
+
+    docker pull gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:$$MARKETPLACE_TOOLS_TAG
+    docker tag gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:$$MARKETPLACE_TOOLS_TAG \
+        gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:local
   waitFor:
-  - Initialize git
-  - Get Kubernetes Credentials
+  - '-'
+
+- id: Pull cloud-marketplace-tools deployer_helm Image
+  name: gcr.io/cloud-builders/docker
+  entrypoint: /bin/bash
+  args:
+  - -exc
+  - |
+    MARKETPLACE_TOOLS_TAG=sha_7c70bcd83a8c490a11f93abfc4ec7a5f39c61ab6
 
-# Wordpress Verification
+    docker pull gcr.io/cloud-marketplace-tools/k8s/deployer_helm:$$MARKETPLACE_TOOLS_TAG
+    docker tag gcr.io/cloud-marketplace-tools/k8s/deployer_helm:$$MARKETPLACE_TOOLS_TAG \
+        gcr.io/cloud-marketplace-tools/k8s/deployer_helm:local
+  waitFor:
+  - '-'
+
+- id: Pull cloud-marketplace-tools dev Image
+  name: gcr.io/cloud-builders/docker
+  entrypoint: /bin/bash
+  args:
+  - -exc
+  - |
+    MARKETPLACE_TOOLS_TAG=sha_7c70bcd83a8c490a11f93abfc4ec7a5f39c61ab6
+
+    docker pull gcr.io/cloud-marketplace-tools/k8s/dev:$$MARKETPLACE_TOOLS_TAG
+    docker tag gcr.io/cloud-marketplace-tools/k8s/dev:$$MARKETPLACE_TOOLS_TAG \
+        gcr.io/cloud-marketplace-tools/k8s/dev:local
+  waitFor:
+  - '-'
+
+#############
+# Wordpress #
+#############
+
+- id: Build Wordpress Deployer
```

Comment:
I prefer the previous way of running make using the dev image. Is the new intention that `cloudbuild.yaml` would list out the steps the user would have to manually run not using `make`? Or is there something inherently not possible to use `make` here?

I find that repeating what `make` does here doesn't make a lot of sense because we're duplicating the logic in 2 places under different languages. I think there are 2 ways a partner can build the application: make, or just custom bash script (or their own make, whatever). In either case, CI should just invoke the same command that the dev would otherwise run on their workstation, potentially with different parameters.